gui-backend/src/test/java/com/rln/gui/backend/implementation/balanceManagement/BalanceCacheTest.java

```diff
@@ -39,9 +39,8 @@ void GIVEN_several_balance_created_events_WHEN_update_cache_THEN_balance_aggrega
         cache.updateBalanceUponCreation(balanceAccount2, cid3);
 
         // THEN
-        Map<String, BigDecimal> balances = cache.getBalance();
-        MatcherAssert.assertThat(balances.get(BalanceTestUtil.IBAN1).doubleValue(), Matchers.is(amount1InAccount1 + amount2InAccount1));
-        MatcherAssert.assertThat(balances.get(BalanceTestUtil.IBAN2).doubleValue(), Matchers.is(amountInAccount2));
+        MatcherAssert.assertThat(cache.getBalance(BalanceTestUtil.IBAN1).doubleValue(), Matchers.is(amount1InAccount1 + amount2InAccount1));
```

Comment:
Looks like the indentation is off in this file.

Reply:
formatted